glossary.md

```diff
@@ -40,18 +40,18 @@ USA -- name --------------> United States of America
 USA -- containedInPlace --> northamerica
 ```
 
-- **Measurement method**
+- **[Measurement method](https://datacommons.org/browser/measurementMethod)**
 
-The source of and technique used for measuring a statistical variable. Describes how a measurement is made, whether by count or estimate or some other approach, and names the group making the measurement. Examples include `BosniaCensus` and `WorldHealthOrganizationEstimates`.
+The technique used for measuring a statistical variable. Describes how a measurement is made, whether by count or estimate or some other approach, and names the group making the measurement. Examples include `BosniaCensus` and [`WorldHealthOrganizationEstimates`](https://datacommons.org/browser/WorldHealthOrganizationEstimates).
```

Comment:
Describes how a measurement is made, whether by count or estimate or some other approach. Names of the group making the measurement may be included to indicate a certain organizational method of measurement is used.